resources/helm/v2.4/istio-telemetry/prometheus/templates/deployment.yaml

```
@@ -128,7 +127,42 @@ spec:
           - mountPath: /etc/istio-certs
             name: istio-certs
           {{- end }}
-
+        - name: config-reloader
+{{- if contains "/" .Values.prometheusConfigReloader.image }}
+          image: "{{ .Values.prometheusConfigReloader.image }}"
+{{- else }}
+          image: "{{ .Values.global.hub }}/{{ .Values.prometheusConfigReloader.image }}:{{ .Values.global.tag }}"
+{{- end }}
+{{- if .Values.global.imagePullPolicy }}
+          imagePullPolicy: {{ .Values.global.imagePullPolicy }}
+{{- end }}
+          args:
+          - --reload-url=http://localhost:9090/-/reload
+          - --config-file=/etc/prometheus/prometheus.yml
+          command:
+          - /usr/bin/prometheus-config-reloader
```

```suggestion
          - /bin/prometheus-config-reloader
```
I got the following error container creation:
```
Error: container create failed: time="2023-04-03T11:33:35Z" level=error msg="runc create failed: unable to start container process: exec: \"/usr/bin/prometheus-config-reloader\": stat /usr/bin/prometheus-config-reloader: no such file or directory"
```